### ISSUE TITLE
 Error when selecting the "sonic-multilingual" model with the language pt in Cartesia results in an error in Pipecat

### ISSUE THREAD

**Issue by J3Mateus** [pipecat-ai/pipecat#513]
**Description:**  
I noticed that when I combine "sonic-multilingual" with the language pt, an error occurs in Pipecat where it doesn't perform the TTS. This happens because, according to the Cartesia documentation, the "pt" language with the "sonic-multilingual" model does not support `add_timestamps` set to True.

**Comment by markbackman** [comment]
This is true. We've asked Cartesia to provide support for more languages. They're hoping to provide support in Q4 of this year. 🤞

**Comment by markbackman** [comment]
Cartesia has added many more languages with timestamps. Closing this issue.

**Comment by kun432** [comment]
seems Cartesia's new model, Sonic-2 supports timestamps for all 15 languages.

https://docs.cartesia.ai/build-with-cartesia/models
